azure-pipelines.yml

```diff
@@ -0,0 +1,65 @@
+trigger:
+  branches:
+    include:
+      - main
+pr:
+  branches:
+    include:
+      - main
+
+pool:
+  vmImage: ubuntu-latest
```

Comment:
I usually prefer to use fixed versions as the vmImage to avoid it changing automatically over time.

In this case I believe it would be ubuntu-20.04